observer.cpp

```diff
@@ -28,11 +28,10 @@ bool operator!=(const all_params& lhs, const all_params& rhs)
 
 
 
-void observer::store_avg_fit_and_env(const simulation& s)
+void observer::store_avg_fit(const simulation& s)
 {
     m_avg_fitnesses.push_back(avg_fitness(s));
     m_var_fitnesses.push_back(var_fitness(s));
-    m_env_values.push_back(get_current_env_value(s));
```

Comment:
Ok,
I am going to add a new issue in which you will have to say the input and optimal value of simulation.